boards/arm/lpcxpresso55s69/CMakeLists.txt

```diff
@@ -9,3 +10,29 @@ if(CONFIG_PINMUX_MCUX_LPC)
   zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
   zephyr_library_sources(pinmux.c)
 endif()
+
+if (CONFIG_BUILD_WITH_TFM)
+	# Set default image versions if not defined elsewhere
+	if (NOT DEFINED TFM_IMAGE_VERSION_S)
+		set(TFM_IMAGE_VERSION_S 0.0.0+0)
+	endif()
+
+	if (NOT DEFINED TFM_IMAGE_VERSION_NS)
+		set(TFM_IMAGE_VERSION_NS 0.0.0+0)
+	endif()
+
+	set(ENV{BL2} False)
```

Comment:
Why setting this as env variable ?
Should this be a Kconfig setting, so that `CONFIG_TFM_BL2` could be used instead ?

Then boards can specify their value for example in: https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0_defconfig

Reply:
will change this, samples will set BL2 False for this board.

Reply:
This could be something like:
```
choice TFM_BL2
        prompt "BL2"
        default TFM_BL2_CONFIG_FILE_DEFAULT

config TFM_BL2_CONFIG_FILE_DEFAULT
        bool "Use TFM BL2 setting from TFM configuration file"

config TFM_BL2_TRUE
        bool "TFM BL2 enabled"

config TFM_BL2_FALSE
        bool "TFM BL2 disabled"

endchoice
```
so that default is to use the TFM BL2 setting from conf file, or specifically enable/disable this feature.

Reply:
This would be a better approach, and if the default is `TFM_BL2_CONFIG_FILE_DEFAULT` we shouldn't need to touch the `mps2_an521` and `nucleo_l552ze_q` targets.

Reply:
@tejlmand @microbuilder made the change requested, please review how this is treated at board level and samples.